themes/docs-new/layouts/shortcodes/ohai_automatic_attribute.md

```diff
@@ -21,38 +21,42 @@ Infra Client run. The most commonly accessed automatic attributes are:
 <td>The platform on which a node is running. This attribute helps determine which providers will be used.</td>
 </tr>
 <tr class="even">
+<td><code>node['platform_family']</code></td>
+<td>The platform family is a Chef Infra specific grouping of similar platforms where cookbook code can often be shared. For example `rhel` includes Red Hat Linux, Oracle Linux, CentOS, and several other platforms that are nearly identical to Red Hat Linux.</td>
```

Comment:
```suggestion
<td>The `platform_family` attribute groups together similar platforms to support sharing cookbook code. For example `rhel` includes Red Hat Linux, Oracle Linux, CentOS, and several other platforms that are nearly identical to Red Hat Linux.</td>
```

Even though the value would be supplied by Infra, InSpec also uses this attribute, so we can't claim it is Infra-specific.

Reply:
InSpec doesn't actually use this value. In a perfect world that would be true, but this is 100% an infra thing.

Reply:
OmO. It calls `platform_family` to match `os.family`.

Reply:
os.family and platform_family do not match.  It's a mess, but we're stuck with it unless one of you has a time machine.